policyengine_us/parameters/gov/usda/summer_ebt/eligibility/categorical.yaml

```diff
@@ -0,0 +1,19 @@
+description: The United States Department of Agriculture requires participation in one of the following programs to qualify for the Summer EBT program.
+
+values:
+  2024-01-01:
+    - snap # Supplemental Nutrition Assistance Program.
+    - tanf # Temporary Assistance to Needy Families.
+    - fdpir # Food Distribution Program on Indian Reservations.
+    - medicaid
+
+metadata:
+  period: year
+  unit: list
+  label: Summer EBT qualifying programs
+  reference:
+    # Programs not mentioned in legal code
```

Comment:
Can we check what the legal code states regarding the eligibility? Is there any thing mentioned (maybe another section)

Reply:
i looked through it again and "surrounding" sections but couldn't find any references to qualifying programs

Reply:
Ok I have updated the note